js/google_ima.js

```diff
@@ -630,7 +637,10 @@ require("../html5-common/js/utils/utils.js");
           {
             //resumeAd will only be called if we have exited fullscreen
             //so this is safe to call
-            this.sharedVideoElement.webkitEnterFullscreen();
+            if (OO.isIosMajorVersion < 10)
```

Comment:
Can we use a separate function to calculate if we support inline playback on iOS? That way we can use this new function here and down below when we exit full screen.